tests/e2e.rs

```diff
@@ -22,15 +22,208 @@ use zitadel_rust_client::v1::{
 	Email, Gender, ImportHumanUserRequest, Phone, Profile, UserType, Zitadel,
 };
```

Comment:
The v1 & v2 use should be updated/verified to match Zitadel operations from `src/zitadel.rs`

Reply:
Yeah, but this isn't particularly high priority. We only *need* to do this once/if the v1 API is deprecated, which doesn't look like it will be happening anytime soon; too many endpoints are still exclusive to v1 for that to be realistic.

So I suggest we put this on the pile of tech debt issues. We should clean up the test suite anyway.

Reply:
Yeah, fair enough.